testing.py

```diff
@@ -754,13 +756,16 @@ def print_database_class(cls, db_connection):
         Be careful of leaving it in code and potentially outputting
         vast tracts of data into your output stream on production.
 
-        Call like this:
-        set_trace()
-        from testing import (
-            DatabaseTest,
-        )
-        _db = Session.object_session(self)
-        DatabaseTest.print_database_class(_db)  # TODO: remove before prod
+        Call like this::
```

Comment:
Code snippets need to go after `::` and a new paragraph. End result:
![Screen Shot 2019-07-31 at 3 28 40 PM](https://user-images.githubusercontent.com/1280564/62245079-babada80-b3ae-11e9-97cc-a10e02612d70.png)